### PR TITLE
implement 2020-10-01: Add 2020-10-01 TX API docs with message about rollup

### PR DESCRIPTION
**Pull request by omeedrj**
[STORY-7731](https://ringrevenue.atlassian.net/browse/STORY-7731)

**NOTE**
This PR is for comparison purposes only. We'll need to follow the README steps to fully implement a `2020-10-01` branch and make that the default branch.

Previous PR: https://github.com/Invoca/developer-docs/pull/238

* Add a new `2020-10-01` API version.
  * Add signal transactions api rollup message for `2019-12-01`.

image

## Checklist

- [ ] Find the [Service owning team](https://docs.google.com/spreadsheets/d/1YF2wuepY5SZTpVEdT9gwhTPhpnDGZeVjywEwvbJS5TQ/edit#gid=0&fvid=1520238175) for these changes, and tag the team as "Reviewers" on this PR
- [x] Test the documentation changes on readthedocs as a private branch
- [ ] ~If changing general content, have agreement on whether to apply to latest version or all versions (if all versions, provide links to the related PRs below)~ N/A

### CODE REVIEW

File: source/api_documentation/transactions_api/_signal_param_table.rst [hunk 1]
```diff
@@ -53,4 +53,3 @@ Most of the fields in this table are now deprecated. See Custom Data & Signal Pa
   * - signal_custom_parameter_3 *(deprecated)*
     - Signal Custom Param 3
     - Up to 255 character string. |rollup_removed_message|
-
```
Comment (peterwilson) [inline source/api_documentation/transactions_api/_signal_param_table.rst:56]: I believe trailing new lines are needed, this should be undone

**Review by peterwilson — CHANGES_REQUESTED** [review]

File: source/api_documentation/transactions_api/_signal_param_table.rst (hunk 1 above)
Comment (omeedrj) [reply source/api_documentation/transactions_api/_signal_param_table.rst:56]: If you look at the actual file there's a trailing newline, I'm removing an extra newline. Github's UI doesn't show the trailing line by default. If the file is missing a trailing newline github's UI will show a red return arrow at the bottom
e.g.
image

Comment (peterwilson) [reply source/api_documentation/transactions_api/_signal_param_table.rst:56]: 🤦‍♂️

**Review by peterwilson — APPROVED** [review]

**Comment by omeedrj** [comment]
Closing this as we're going to make the `2020-10-01` branch the new default branch for the repo

**Pull request closed without merging**